src/cryptoadvance/specter/templates/wallet/settings/wallet_settings.jinja

```diff
@@ -180,28 +181,32 @@
 				<div class="row aligned">
 					<textarea name="address_labels_data" 
 					id="address_labels_data" 
-					placeholder="{{ _('You can paste or drag & drop here: Electrum JSON file, Specter addresses CSV or Specter JSON file.') }}"  
-					ondrop="dropHandler(event, 'address_labels_data');" 
-					ondragover="dragOverHandler(event, 'address_labels_data');"
-					ondragleave="dragLeaveHandler(event, 'address_labels_data');" 
-					ondragexit="dragExitHandler(event, 'address_labels_data');" 
-					ondragend="dragEndHandler(event, 'address_labels_data');"></textarea>
+					placeholder="{{ _('You can paste and drag & drop here') }}
+					{{ _('(Electrum JSON file, Specter addresses CSV or Specter JSON file)') }}"  
+					ondrop="dropHandler(event);" 
+					ondragover="dragOverHandler(event);"
+					ondragleave="dragLeaveHandler(event);" 
+					ondragexit="dragExitHandler(event);" 
+					ondragend="dragEndHandler(event);"
+					style="margin-bottom: 20px; height: 150px; font-size: 13px;">
+					</textarea>
```

Comment:
Could you also create a template for this textarea (+the javascript code that makes the `ondrop`, ... work)?    Then the `psbt-uploader`  could have the same drag&drop effects.

Reply:
We can do this in a separate issue / PR, good idea!

Reply:
I made a new issue for that: https://github.com/cryptoadvance/specter-desktop/issues/1704